test/contract_sophia_bytecode_aevm_SUITE.erl

```diff
@@ -332,28 +332,28 @@ oracles(_Cfg) ->
     Env0 = initial_state(#{}),
     Env1 = create_contract(101, Code, "()", Env0),
     QFee = 100,
-    {101, Env2} = successful_call(101, word, registerOracle, "(101, 3, "++ integer_to_list(QFee) ++", 10)", Env1),
-    {Q, Env3}   = successful_call(101, word, createQuery, "(101, \"why?\", "++ integer_to_list(QFee) ++", 10, 11)", Env2, #{value => QFee}),
+    {101, Env2} = successful_call(101, word, registerOracle, "(101, 3, "++ integer_to_list(QFee) ++", (0, 10))", Env1),
```

Comment:
I understand from aeso_icode that 0 means relative. Shall this (and 1 for fixed) be documented in https://github.com/aeternity/protocol/pull/178 e.g. in section "Data memory layout"?

Reply:
https://github.com/aeternity/protocol/pull/178/commits/c22456366c7debd9ba53b169b92ed8a9b3e6509b